pypeit/scripts/chk_noise_2dspec.py

```diff
@@ -0,0 +1,260 @@
+"""
+This script explores the noise in a slit or spectrum
+for one of the outputs of PypeIt
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+import imp
+import os
+
+import numpy as np
+
+import os
+from matplotlib import pyplot as plt
+from astropy.stats import sigma_clip, mad_std
+from astropy.modeling.models import Gaussian1D
+from astropy.table import Table
+import argparse
+
+from sympy import im
```

Comment:
Is this used?  I.e., does this add a new dependency?

Reply:
removed